nym-vpn-core/nym-vpn-cli/src/main.rs

```diff
@@ -20,13 +20,20 @@ use nym_vpn_lib::nym_config::defaults::{setup_env, var_names};
 
 const CONFIG_DIRECTORY_NAME: &str = "nym-vpn-cli";
 
-pub fn setup_logging() {
-    let filter = tracing_subscriber::EnvFilter::builder()
+pub(crate) fn setup_logging(args: &CliArgs) {
+    let mut filter = tracing_subscriber::EnvFilter::builder()
         .with_default_directive(tracing_subscriber::filter::LevelFilter::INFO.into())
         .from_env()
         .unwrap()
         .add_directive("hyper::proto=info".parse().unwrap())
         .add_directive("netlink_proto=info".parse().unwrap());
+    if let Commands::Run(run_args) = &args.command {
+        if run_args.enable_wireguard {
```

Comment:
Off topic: should it be `--wireguard-mode` instead of `--enable-wireguard`?

Reply:
Sounds good to me